### ISSUE TITLE
Crash report "java.lang.NullPointerException: Attempt to read from field 'android.view.ViewGroup com.github.adamantcheese.chan.controller.Controller.view' on a null object reference""

### ISSUE THREAD

**Issue by K1rakishou** [Adamantcheese/Kuroba#618]
UUID: 9e65b4b73864f64cd6723a0c5f3b0d00
Build flavor: stable
Version name: v4.11.0
OS info: Android 9, sdk version: 28
Report description: No title
Logs:
```
Called from unhandled exception handler.
java.lang.NullPointerException: Attempt to read from field 'android.view.ViewGroup com.github.adamantcheese.chan.controller.Controller.view' on a null object reference
	at com.github.adamantcheese.chan.controller.ui.NavigationControllerContainerLayout.setTopControllerTranslation(NavigationControllerContainerLayout.java:319)
	at com.github.adamantcheese.chan.controller.ui.NavigationControllerContainerLayout.access$400(NavigationControllerContainerLayout.java:38)
	at com.github.adamantcheese.chan.controller.ui.NavigationControllerContainerLayout$1.run(NavigationControllerContainerLayout.java:299)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:949)
	at android.view.Choreographer.doCallbacks(Choreographer.java:761)
	at android.view.Choreographer.doFrame(Choreographer.java:693)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:935)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6669)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

**Comment by K1rakishou** [comment]
Maybe it was related to tollbar issues and is already fixed.

**Comment by Adamantcheese** [comment]
This is on stable, so likely not.

**Comment by K1rakishou** [comment]
Oh, yes.

**Comment by Adamantcheese** [comment]
Likely resolved as of #611.
